package.json

```diff
@@ -17,7 +17,7 @@
     "release-beta": "lerna publish --dist-tag beta",
     "clean": "rimraf './packages/*/dist'",
     "generate": "plop",
-    "generate:package": "plop package && yarn plop docs",
+    "generate:package": "plop package && yarn plop docs && prettier --write \"./packages/tsconfig.json\"",
```

Comment:
Run prettier on the updated `/packages/tsconfig.json`

Reply:
I think it makes sense. This way we don't rely on our committed templates being properly prettified.

Reply:
👀  a bit more and you don't have access to the package's name here.